openapi/golem-service.yaml

```diff
@@ -2929,9 +2929,32 @@ components:
       - componentId
       - workerName
       - response
+    GolemWorkerBindingWithTypeInfo:
+      type: object
+      properties:
+        componentId:
+          $ref: '#/components/schemas/VersionedComponentId'
+        workerName:
+          type: string
+        idempotencyKey:
+          type: string
+        response:
+          type: string
+        responseMappingInput:
+          $ref: '#/components/schemas/RibInputTypeInfo'
+        workerNameInput:
+          $ref: '#/components/schemas/RibInputTypeInfo'
+        idempotencyKeyInput:
+          $ref: '#/components/schemas/RibInputTypeInfo'
+      required:
+      - componentId
+      - workerName
+      - response
+      - responseMappingInput
+      - workerNameInput
```

Comment:
responseMappingInput, workerNameInput - those look like are new required attributes, which will break backward compatibility

Reply:
Its on the response and not on the input

Reply:
How can adding a required field on response of every API breaks compatibility?

Reply:
No change is done on the input data types (HttpApiDefinitionRequest or HttpApiDefinition)

Reply:
I know, I was discussing same situation with createdAt fields also in respinses only, and general opinion was it shold be optional